README.md

```diff
@@ -6,7 +6,7 @@ Small, fast and accessibility-first React carousel library with easily customiza
 
 ![Nuka Carousel Animated Example](https://i.imgur.com/UwP5gle.gif)
 
-> If you are looking for v4 documentation, you can find it [here](https://github.com/FormidableLabs/nuka-carousel/blob/main/V4-DOCUMENTATION.md)
```

Comment:
Could we not delete the link but replace it with this one https://www.npmjs.com/package/nuka-carousel/v/4.8.4 , as a lot of people still use it. Thanks for raising this PR

Reply:
Ok done raised another PR with this https://www.npmjs.com/package/nuka-carousel/v/4.8.4